### PR TITLE
Perf/william/tls buffer defaults

### PR DESCRIPTION
**Pull request by qzhuyan**
-  perf(TLS): set default sndbuf and recbuf to 4K
- enable TLS process hibernation.

### CODE REVIEW

File: CHANGES-4.3.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ File format:
   Change log ends with a list of GitHub PRs
 
 ## v4.3.20
```
Comment (zmstone) [inline CHANGES-4.3.md:13]: 4.3.20 is already released
please rebase on latest main-v4.3 and move this to 4.3.21

**Comment by zmstone** [comment]
This change requires a justification with performance regression.

**Review by thalesmg — APPROVED** [review]

File: etc/emqx.conf [hunk 2]
```diff
@@ -1651,6 +1651,12 @@ listener.ssl.external.reuseaddr = true
 ## Value: true | false
 ## listener.ssl.external.gc_after_handshake = false
 
+## hibernate the SSL process after idling for amount of time
+## Default: undefined (off)
+##
+## Value: Duration
+listener.ssl.external.hibernate_after = 5s
```
Comment (thalesmg) [inline etc/emqx.conf:1658]: Maybe keep commented to use the default (current behavior)?

Comment (qzhuyan) [reply etc/emqx.conf:1658]: yes I think it is good idea

**Comment by zmstone** [comment]
Deserves two PRs
Each with detailed CHANGES update.

**Comment by qzhuyan** [comment]
replaced with #9007 and #9005

**Pull request closed without merging**